pages.id/common/airmon-ng.md

```diff
@@ -1,6 +1,6 @@
 # airmon-ng
 
-> Nyalakan mode pengawasan pada perangkat jaringan nirkabel/wireless
+> Nyalakan mode pengawasan pada perangkat jaringan nirkabel/wireless.
```

Comment:
```suggestion
> Nyalakan mode pengawasan pada perangkat jaringan nirkabel/*wireless*.
```
I see some Indonesian translations are using italics to highlight foreign words, but is that allowed by our style guide?

Reply:
Hi, I don't think we should add it to the style guide (the other pages must be fixed) as throughout our clients we don't support bold and italics in pages so adding it to Indonesian pages alone would lead to confusion and failing checks too.